### PR TITLE
Add more loading messages

### PR DESCRIPTION
**Pull request by SuperSmay**
Goji you were too slow

### CODE REVIEW

File: Assets/__Scripts/Localization/Localization.asset [hunk 1]
```diff
@@ -14,7 +14,7 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   OverwriteLocalizationText: 0
   OverwriteLocalizationTextID: 166
-  loadingMessages:
+  LoadingMessages:
```
Comment (SuperSmay) [inline Assets/__Scripts/Localization/Localization.asset:17]: Unity why

File: Assets/__Scripts/Localization/Localization.asset [hunk 2]
```diff
@@ -442,3 +442,258 @@ MonoBehaviour:
...
+    of Technicolour\xA9 lightwaves, to bring your eyes to their knees! Now, with
+    something as brilliant and Nobel Peace Price worthy as the Chroma Suite\u2122,
+    specialized software needs to be created to handle these beasts of technological
+    engineering. \nThat's where ChroMapper\xAE comes into play! ChroMapper\u2122
+    makes it easy for the everyday Joe to use the power of Chroma\xA9 RGB\u2122 to
+    create lightshows and maps that are far superior to their duochromatic counterparts!
+    Reality Check Through The Skull? Slap some Chroma on that mofo and you've got
+    yourself a map worthy of Cube Community! So, what are you waiting for? Stop living
+    in your sad two-color world and join us enlightened individuals with the power
+    of Chroma\xAE RGB and ChroMapper\u2122! "
```
Comment (Caeden117) [inline Assets/__Scripts/Localization/Localization.asset:491]: shut

File: Assets/__Scripts/Localization/Localization.asset [hunk 3]
```diff
@@ -442,3 +442,258 @@ MonoBehaviour:
...
+    report crashes, the crash report. That report?
+  - 'why are we still here
+
+    just to suffer'
+  - Chro with your Bro
+  - Generating Jason files...
+  - Now available on the Gameboy Color
+  - what happened to the 4th stan
+  - used by leading experts in 'epileptronics'
+  - used by leading experts in 'epileptronics'
```
Comment (GalaxyMaster2) [inline Assets/__Scripts/Localization/Localization.asset:655]: the message so nice you added it twice

Comment (SuperSmay) [reply Assets/__Scripts/Localization/Localization.asset:655]: whoops i knew there was at least one

**Pull request merged**